packages/client/src/features/modal/components/SendBookingsLinkDialog/__tests__/SendBookingsLinkDialog.test.tsx

```diff
@@ -27,8 +29,10 @@ jest
   .mockImplementation(mockSendBookingsLink as any);
 
 const mockDispatch = jest.fn();
+const mockSelector = jest.fn();
```

Comment:
I don't like this and would rather test this `mockSelector` against an instance of the store (using `renderWithRedux`) and passing a store populated with some data, but this won't be necessary if you move the display name logic to the thunk itself.